KIF Tests/ScrollViewTests.m

```diff
@@ -16,6 +16,7 @@ @implementation ScrollViewTests
 
 - (void)beforeEach
 {
+    [[tester class] setTestActorAnimationsEnabled:YES];
```

Comment:
It would be best to set this back to YES in `afterEach` instead. You could assert that it is YES at the start of the tests if you want to make sure the default value is correct. When the entire test suite is run, if the last test that is executed happens to be one with animations disabled, it might pollute other tests.

Reply:
Where would you assert if the flag is YES? I put it on `beforeEach` in the scrollView/tableview/collectionView tests, but let me know if you were thinking of a different place

Reply:
That is perfect